src/Lib.hs

```diff
@@ -10,4 +10,4 @@ someFunc :: IO ()
 someFunc = do
     content <- readFile "./app/resources/code.yamil"
     let result = parse (P.parseProgram) "./app/resources/code.yamil" content
-    print result
+    print result
```

Comment:
What is the change of this class? I same the line of code but without the extra empty line.